generic/nstress.py.data/nstress.yaml

```diff
@@ -10,3 +10,4 @@ test: !mux
         nmem64:
                 testcase: "nmem64"
                 memchunk: "2047"
+
```

Comment:
@PraveenPenguin white space needs to be removed please

Reply:
pushed the changes please have a look